lib/fhir_models/bootstrap/model.rb

```diff
@@ -6,8 +6,8 @@
 module FHIR
   class Model
     extend FHIR::Deprecate
-    def initialize(hash = {})
-      from_hash(hash)
+    def initialize(hash = {}, version = self.class.name.split('::')[1])
```

Comment:
I think I'm a little confused by your question.   Are you proposing that instead of using const_get, have a wrapper method somewhere in `models.rb` or elsewhere that is more human readable that gives the version?  Or something else?  

I wanted to allow a way to force a versioned model, as I don't think you can just infer it from the resource's representation (at least not always).  But if I'm understanding correctly, the issue isn't version as a parameter, it'll just be easier developmentally going forward if this is more human readable?

Reply:
`FHIR::Model` itself is never actually instantiated, right? Only the generated classes that inherit from `Model` are instantiated. Those models are each generated for a specific version of FHIR, so they could report their own version rather than having to inspect the class name (which I don't think will be reliable if someone subclasses a FHIR model in their own namespace).

If a model class knows all of its fields, their types and cardinalities, why can't it also just say know version of FHIR it was generated from?